robot_ws/src/cloudwatch_robot/package.xml

```diff
@@ -1,39 +1,32 @@
-<?xml version="1.0"?>
-<package format="2">
+<?xml version='1.0' encoding='utf-8'?>
+<package format="3">
   <name>cloudwatch_robot</name>
   <version>1.1.0</version>
   <description>
   AWS RoboMaker robot package that provides a sample application for integrating AWS CloudWatch with ROS.
   </description>
-  <license>MIT</license>
   <author email="ros-contributions@amazon.com">AWS RoboMaker</author>
   <maintainer email="ros-contributions@amazon.com">AWS RoboMaker</maintainer>
-  <buildtool_depend>catkin</buildtool_depend>
-  <depend>rospy</depend>
-  <depend>roscpp</depend>
-
+  <license>MIT</license>
+  <buildtool_depend>ament_python</buildtool_depend>
```

Comment:
I don't think ament_python is a buildtool option. we probably don't need a buildtool here? (I've seen some python packages use `ament_package`)